Posodobljen postopek generiranja IBKR poročila (#49)
jamsix/ib-edavki: README.md

### Izvoz poročila v platformi InteractiveBrokers

1. V meniju **Performance & Statements** pod **Reports** odpri **Flex Queries**

> 1. Desno od napisa Activity Flex Query kliknite + za kreiranje novega poročila.

Reviewer: ```suggestion
1. Desno od napisa Activity Flex Query klikni `+` za kreiranje novega poročila.
```

### Izvoz poročila v platformi InteractiveBrokers

> 1. V meniju **Reports** odpri **Flex Queries**

Reviewer: Meni namesto Reports piše *Performance & Statements* - gledam kaj drugega?

![CleanShot 2022-02-12 at 19 41 25@2x](https://user-images.githubusercontent.com/3621/153724018-2ec8cff1-a97c-4e80-8997-3595db1449b3.png)

Author: Ja, sem mislil da je že prej tako pisalo in da se je un Reports nanašal na podfolder. Sem še malo dodatno dopisal.

1. Desno od napisa Activity Flex Query kliknite + za kreiranje novega poročila.
1. Vpiši poljuben **Query Name**
1. Pod **Sections** klikni na **Account Information**. Izberi **IB Entity** in **Account ID** in potem na dnu Save.
1. Pod **Sections** klikni na **Trades**. Pod Options označi **Executions** in **Closed Lots**. Izberi vse stolpce (**Select All**).
1. Pod **Sections** klikni na **Corporate Actions**. Izberi vse stolpce (**Select All**).
1. Pod **Sections** klikni na **Cash Transactions**. Pod Options označi **Dividends**, **Payment in Lieu of Dividends**, **Withholding Tax** in **Broker Interest Received**. Izberi vse stolpce (**Select All**).
1. Pod **Sections** klikni na **Financial Instrument Information**. Izberi vse stolpce (**Select All**).
1. Vse ostale nastavitve pusti tako kot so(datumske nastavitve bomo spreminjali ob zaganjanju poročila).
1. Na dnu klikni **Continue** in nato **Create**, za tem bo treba še enkrat **Ok** in bomo nazaj na oknu **Flex Queries**.

> 1. V pogledu **Reports > Flex Queries** se je pod **Activity Flex Query** pojavila nova vrstica z vašim novim poročilom.

Reviewer: ```suggestion
1. V pogledu **Reports > Flex Queries** se je pod **Activity Flex Query** pojavila nova vrstica s tvojim novim poročilom.
```

> 1. Poročilo zaženemo z klikom na puščico desno v poleg imena poročila. Tam sedaj spremenimo **Period** na **Custom Date Range** in spodaj ta rang nastavimo na prvi in zadnji dan leta za katerega generiramo. Potem pa samo poženi poročilo in dobiš datoteko.

Reviewer: ```suggestion
1. Poročilo zaženeš s klikom na puščico v desno poleg imena poročila. Tam sedaj spremeniš **Period** na **Custom Date Range** in spodaj ta rang nastaviš na prvi in zadnji dan leta za katerega generiraš (če katerega od datumov ni možno izbrati, beri spodnjo opombo glede Brexita). Potem samo poženi poročilo in dobiš datoteko.
```

1. Ponovi postopek za vsako leto trgovanja, če si trgoval v letih 2016, 2017 in 2018, generiraj 3 reporte, po enega za vsako leto. Za pravilen izračun tujega davka na dividende (Withholding Tax) je potrebno generirati tudi report tekočega leta, saj so nekateri obračuni poročani za nazaj.

> Ker se z 2021 zaradi Brexita večino evropskih IBKR računov premaknilo(oziroma realno so se računi ukinili in novi kreirali) iz Londona v eno izmed evropskih podružnic lahko naletite na težave pri izvozu podatkov. Na strani **Reports** je desno od tega v modrem še številka vašega računa, če jo kliknete se vam odpre meni kjer lahko izbirate za kateri račun želite poročila, po defaultu sicer ne kaže ukinjenih računov, vendar če kliknete na filter lahko to spremenite da se pokaže tudi ukinjen račun. Potem pa obkljukate še tistega in kliknete **Continue**. Sedaj boste poleg naslova **Reports** videli dve številki računov, in tudi podatki bodo iz obeh.

Reviewer: ```suggestion
Ker se je z 2021 zaradi Brexita večino evropskih IBKR računov premaknilo (oziroma realno so se računi ukinili in novi kreirali) iz Londona v eno izmed evropskih podružnic lahko naletiš na težave pri izvozu podatkov. Na strani **Reports** je desno od tega v modrem še številka tvojega računa. Če jo klikneš se odpre meni kjer lahko izbiraš za kateri račun želiš poročila, po defaultu sicer ne kaže ukinjenih računov, vendar če klikneš na filter lahko to spremeniš, da se pokaže tudi ukinjen račun. Potem pa obkljukaš še tistega in klikneš **Continue**. Sedaj boš poleg naslova **Reports** videl dve številki računov, in tudi podatki bodo iz obeh.
```
